glib/src/closure.rs

```diff
@@ -118,6 +118,22 @@ impl Closure {
     }
 }
 
+pub trait ToOptionValue {
```

Comment:
Maybe should be named `ToClosureReturn` or `ToClosureReturnValue`?

Something like this is basically what I was thinking of in https://github.com/gtk-rs/gtk-rs-core/issues/337 btw @bilelmoussaoui , just the other way around